doc/api/cli.md

```diff
@@ -144,6 +144,9 @@ added: v12.12.0
 
 Enable experimental Source Map V3 support for stack traces.
 
+Note, that `--enable-source-maps` does not work in conjunction with the
```

Comment:
We avoid using "note" in the docs. How about:

Overriding `Error.prepareStackTrace` is ignored when `--enable-source-maps` is enabled.